### DISCUSSION TITLE
Possible to wrap on indent for plaintext?

### DISCUSSION THREAD

**Discussion by kinghezy** [gsantner/markor#2101] [Q&A] [answered]
In notepad++ for plain text, the text will wrap at the indent (see below). Is there any setting in Markor that allows the same in plain-text?  (I checked the FAQ , settings, and searched github to see if this has come up. I didn't see this question addressed.)

![image](https://github.com/gsantner/markor/assets/65113095/768f4fe0-24ef-4004-8d4f-e2c5c31651bf)

**Comment by gsantner** [comment, marked as answer]
There is no such functionality in Markor.
